src/Makevars

```diff
@@ -1 +1,2 @@
-PKG_LIBS= -lGL -lGLU -lSDL -LSDmain
+PKG_CXXFLAGS = -I/usr/local/Cellar/sdl2/2.0.16/include/ -I/opt/X11/include
+PKG_LIBS= -lSDL2
```

Comment:
After some more digging, it looks like [SDL2 wraps around OpenGL](https://www.libsdl.org/release/SDL-1.2.15/docs/html/guidevideoopengl.html), so linking OpenGL and GLUT here is unnecessary. Thanks for pointing that out, @Jean-Romain.